sensor_array/analysis/execute_sample_analysis.py

```diff
@@ -6,37 +6,6 @@
 import sample_analysis as sa
 
 
-# ========= TEST SECTION ==========
-"""
-To Do list
-
--Write a function which reads the results from predicted breath Samples
--Visualize results in a way which makes sense
-    - All samples (1-48), separated by component (4 plots total)
-    - Same for true comp vs. simulated comp.
--Decide healthy/diseased cutoff & create a convolution matrix
--Generally, look for trends in data
-    -When do we over/under predict ammonia (high/low CO2?)
-    -Minimum and maximum deviation
-    -After what percent cutoff do we stray from true comp
-    -Should 'fraction_to_keep' be varied
-        Since points start to cluster you can relax how many points to keep
-    -Try other arrys/array Sizes
-    -Vary Corys method to weight results by certain gas
-
-"""
-
-"""
-Overall Procedure:
-    1. Evaluate / normalize element probability
-    2. Evaluate / normalize array probability
-    3. Re-order compositions by probability (high to low)
-    4. Keep top points / create new grid
-        4.1 Remove duplicate Points
-        4.2 Reformat as dict
-    5. Repeat from Step 1
-"""
```

Comment:
In general I would call this commit something like 'removed stale comments' not 'Cleaned up execute script' since cleaning up could also be code changes.

Reply:
Fair enough. I'll try to be more exact with future commit messages.